pkg/apis/workloads/v1alpha1/console_types.go

```diff
@@ -52,11 +62,17 @@ type ConsoleTemplate struct {
 }
 
 type ConsoleTemplateSpec struct {
-	User                     string                 `json:"user"`
-	Template                 corev1.PodTemplateSpec `json:"template"`
-	DefaultTimeoutSeconds    int                    `json:"defaultTimeoutSeconds"`
-	MaxTimeoutSeconds        int                    `json:"maxTimeoutSeconds"`
-	AdditionalAttachSubjects []rbacv1.Subject       `json:"additionalAttachSubjects"`
+	User     string                 `json:"user"`
+	Template corev1.PodTemplateSpec `json:"template"`
+
+	// Default time, in seconds, that a Console will be created for
+	// +kubebuilder:validation:Maximum=604800
+	DefaultTimeoutSeconds int `json:"defaultTimeoutSeconds"`
+
+	// Maximum time, in seconds, that a Console can be created for
+	// +kubebuilder:validation:Maximum=604800
```

Comment:
Could add minimums on these fields too?

Reply:
Good point, I'll do this. The core k8s types don't appear to use unsigned ints anywhere, so we followed the convention of using plain int on this field, but we definitely don't want clients to submit negative values.